mutalyzer_api/api.py

```diff
@@ -2,7 +2,7 @@
 
 
 def main():
-    app.run(debug=True)
+    app.run(host="0.0.0.0", debug=True)
```

Comment:
🔧 probably best to turn debug off for prod but not a blocker since it won't be exposed publicly